Extend/Model/Payment_Gateway.php

```diff
@@ -172,9 +165,7 @@ public function makeTransaction(float $fAmount, Order $oOrder): Response
             $this->_addBasketInfo($oTransaction, $oBasket);
         }
 
-        $oTransaction->setNotificationUrl($sShopUrl
-            . 'index.php?cl=wcpg_notifyhandler&fnc=handleRequest&pmt='
-            . Payment_Method::getOxidFromSDKName($oPaymentMethod->getTransaction()->getConfigKey()));
+        $oTransaction->setNotificationUrl(NotifyHandler::getNotificationUrl($oPaymentMethod));
```

Comment:
please typhint what oTransaction is. And you deliver the getNotificationUrl an wrong class or your phpdoc is wrong

Reply:
👍